index.Rmd

```diff
@@ -286,7 +290,7 @@ In order to achieve these objectives, you should try to follow a few good practi
     - Consistent placement of curly braces.
 * [Make your own packages](#__optional__Writing_packages_).
```

Comment:
This link is not working but I am not sure how to fix it. Goes to the start of the tutorial instead of going to the section on writing packages.